src/modules/shared/window.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+
+function _window(): any {
```

Comment:
I prefer the naming conventions used in the answer on the top of this page: http://stackoverflow.com/questions/34177221/angular2-how-to-inject-window-into-an-angular2-service

- Using "service" in the name of the service
- Not using the underscore for the method.
- Making `getWindow` a verb, instead of `_window()`

```
import { Injectable } from '@angular/core';

function getWindow(): any {
  return window;
}

@Injectable()
export class WindowRefService {
  get nativeWindow(): any {
    return getWindow();
  }
}
```